src/pymor/parallel/ipython.py

```diff
@@ -169,6 +176,10 @@ def _map(self, function, chunks, **kwargs):
     def _remove_object(self, remote_id):
         self.view.apply(_remove_object, remote_id)
 
+    def _update_defaults(self):
+        self._apply(defaults.set_defaults, defaults.get_defaults(_user=True, _file=False, _code=False))
```

Comment:
I think at least `_file` should be set to `True`. Do you have any reasons for not doing that?

Reply:
Every process (IPython and MPI) has to execute `pymor.__init__` where the defaults from file are loaded. As long as everyone has access to the file, there is no need to update them.
Are IPython Worker Pools supposed to run without a shared file system? Wouldn't that break `pymor.parallel.ipython.___enter__`: `client[:].apply_sync(os.chdir, os.getcwd())`? (line 107)

Reply:
This assumes that the defaults are stored within `pymor_defaults.py`. You can also manually load defaults from wherever using `load_defaults_from_file`.

Yes, IPythonPool is supposed to also work without a shared filesystem. So you are right that the mentioned could should be better guarded with `try: ... except: ...`.